pkg/tinydtls/patches/0006-crypto-only-one-HMAC-context-is-required.patch

```diff
@@ -0,0 +1,95 @@
+From 49de0a5763e5ced05f62b2539a63e90b9a439c18 Mon Sep 17 00:00:00 2001
+From: "Martine S. Lenders" <m.lenders@fu-berlin.de>
+Date: Tue, 29 Mar 2022 12:06:00 +0200
+Subject: [PATCH] crypto: only one HMAC context is required
```

Comment:
Is there an upstream PR for this?
All other patches have been merged by now.

Reply:
From https://github.com/RIOT-OS/RIOT/pull/17878#issue-1183424430

> ### Issues/PRs references
> 
> Patch provided upstream in [eclipse/tinydtls#130](https://github.com/eclipse/tinydtls/pull/130)

Reply:
(it was all squashed into one commit there)